Sources/WormholySwift/Extensions/UIViewController+Ext.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Wormholy.swift
+//  Wormholy
+//
+//  Created by Paolo Musolino.
+//  Copyright © 2018 Wormholy. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+extension UIViewController {
+
+    static func currentViewController(_ viewController: UIViewController? = UIApplication.shared.windows.filter(\.isKeyWindow).first?.rootViewController) -> UIViewController? {
+        guard let viewController else { return nil }
+
+        if let viewController = viewController as? UINavigationController {
+            if let viewController = viewController.visibleViewController {
+                return currentViewController(viewController)
+            } else {
+                return currentViewController(viewController.topViewController)
+            }
```

Comment:
In general, I would add a comment here that clarifies every condition as a future reminder.